spec/system/feedback_fish_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Feedback fish" do
+  let(:local_authority) { create(:local_authority, :default) }
+  let(:assessor) { create(:user, :assessor, local_authority: local_authority) }
+
+  before do
+    sign_in assessor
+  end
+
+  context "when FEEDBACK_FISH_ID env variable is set" do
+    before do
+      ENV["FEEDBACK_FISH_ID"] = "randomid123"
+    end
+
+    it "is displayed with a reference to the user's email" do
+      visit root_path
+
+      within find("a[data-feedback-fish][data-feedback-fish-userid='#{assessor.email}']") do
+        expect(page).to have_content "feedback"
+      end
+    end
+  end
```

Comment:
I don't understand this - could it not be like the test below and just check for an email? I might be misunderstanding

Reply:
Attached some screenshots.

The email address is just sent as metadata so not actually testing visually for this

Reply:
Just want to test that the correct html is formed for this widget